tests/testthat/test-get_summary.R

```diff
@@ -4,4 +4,200 @@
 #' T1.3 An error when `survfit_object` is a tibble
 #' T1.4 An error when `survfit_object` is a data.table
 #' T1.5 An error when `survfit_object` is a random object
-#' T1.6 An error when `survfit_object` is NULL
\ No newline at end of file
+#' T1.6 An error when `survfit_object` is NULL
+#' T2. The function accepts a survival object
+#' T2.1 An error when `statlist` is NULL
+#' T2.2. An error when `statlist` is a random string
```

Comment:
An error when the statlist contains non-allowed strings eg "blah"

+ No error when the statlist contains ... or .... or ...